docs/data-sources/kibana_security_role.md

```diff
@@ -0,0 +1,96 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "elasticstack_kibana_security_role Data Source - terraform-provider-elasticstack"
+subcategory: ""
```

Comment:
Can we create docs [templates](https://github.com/elastic/terraform-provider-elasticstack/blob/main/templates/resources/kibana_alerting_rule.md.tmpl) for these new resources with the subcategory set to Kibana.